openquake/server/tests/engine_test.py

```diff
@@ -1,112 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
The test below was deemed more annoying than useful. At the end. it was only checking the format of some display output, which is annoying (if I change the display the test breaks).